docs/src/pages/pictures/shape.md

```diff
@@ -20,7 +20,10 @@ val basicShapes =
 
 This program gives the output below.
 
-@:image(basic-shapes.png)
+@:image(basic-shapes.png) {
+  alt = A circle, square, and triangle
```

Comment:
I think you need to quote this string and the one below it, as the commas seem to throw off the parser.

Reply:
Done, removed the commas